src/metabase/driver/sql_jdbc/sync/describe_database.clj

```diff
@@ -193,6 +193,14 @@
         :else
         nil))
 
+(defn- add-table-properties [driver database tables]
+  (if (driver/database-supports? driver :sync-row-count database)
+    (let [schema+table->row-count (driver/schema+table->row-count driver database)]
+      (eduction (map (fn [{:keys [schema name] :as table}]
+                       (assoc table :properties {:row-count (get schema+table->row-count [schema name])})))
```

Comment:
Can we rename this to `:estimated-row-count`? I'm worried about future features not taking these numbers with sufficient salt.

Reply:
yep, renamed in https://github.com/metabase/metabase/pull/39494/commits/1ef0ec58be8a5ae875ec3c0a42cb13d219d6b94e